client/profile/self-profile-overlay.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { UseTransitionProps } from 'react-spring'
+import { Popover, useAnchorPosition } from '../material/popover'
+import { defaultSpring } from '../material/springs'
+import { Actions, Header, PopoverContents, StyledAvatar, Username } from './profile-overlay-content'
+
+interface SelfProfileOverlayProps {
+  open: boolean
+  username: string
+  anchor: HTMLElement | null
+  children: React.ReactNode
+  onDismiss: () => void
+}
+
+const VERTICAL_TRANSITION: UseTransitionProps<boolean> = {
+  from: { opacity: 0, scaleY: 0.5 },
+  enter: { opacity: 1, scaleY: 1 },
+  leave: { opacity: 0, scaleY: 0 },
+  config: (item, index, phase) => key =>
+    phase === 'leave' || key === 'opacity' ? { ...defaultSpring, clamp: true } : defaultSpring,
+}
+
+export function SelfProfileOverlay(props: SelfProfileOverlayProps) {
+  const { username, children, open, onDismiss, anchor } = props
+  const [, anchorX, anchorY] = useAnchorPosition('left', 'top', anchor ?? null)
```

Comment:
These positions (and the origins below) don't seem right for this? It's extending upwards from the bottom of the screen, so you probably want center, top for the anchor, and center, bottom for the origin

Reply:
On second thought this probably identical given that the widths are the same, that's fine